group_vars/galaxy.yml

```diff
@@ -742,6 +743,9 @@ base_app_main: &BASE_APP_MAIN
   use_heartbeat: "True"
   heartbeat_interval: "-1"
   heartbeat_log: "{{ galaxy_log_dir }}/heartbeat_{server_name}_{pid}.log"
+  # Potentially fix the "could not recover job"
+  # xref https://github.com/galaxyproject/galaxy/pull/6924/
```

Comment:
Nice. Is this also in 18.09?

Reply:
Yes, the default will change, so we could potentially remove the variable we add temporarily.